### ISSUE TITLE
Missing treasury event types?

### ISSUE THREAD

**Issue by thomasmost** [kgajera/stripe-event-types#5]
Looks like we are missing the treasury event types

```ts
/// <reference types="stripe-event-types" />

import express from "express";
import Stripe from "stripe";

export const webhook = async (
    req: express.Request,
    res: express.Response
  ) => {
    const event = req.body as Stripe.DiscriminatedEvent;
    if (event.type === "treasury.outbound_payment.posted") { // causes a ts(2367) "no overlap" error

    }
    if (event.type === "treasury.outbound_payment.returned") { // causes a ts(2367) "no overlap" error

    }
  }
```

I'd be happy to jump in and help implement this if you can point me in the right direction

**Comment by kgajera** [comment]
Thanks for reporting this! 

The types are being generated from the events listed in their docs and I don't see any treasury events listed here: https://stripe.com/docs/api/events/types

Do you know if Stripe has any docs on these events?

**Comment by thomasmost** [comment]
They're there, below Issuing and Terminal:
https://stripe.com/docs/api/treasury/financial_accounts

Scratch that, I see what you mean—they're not in the event list. Let me ask the Stripe team!

**Comment by thomasmost** [comment]
I love it when a plan comes together 😉 Looks like they just updated the list of event types!

**Comment by thomasmost** [comment]
Ugh, apparently since treasury is not 'self-serve' you have to be logged in to an enabled account for those events to be included. Never mind... not sure there's anything to be done here, unless we can provide Puppeteer with credentials for a Stripe account enabled for Treasury in test...

**Comment by kgajera** [comment]
I contacted Stripe to get access to Treasury but had no luck with that.

Their [OpenAPI Spec](https://raw.githubusercontent.com/stripe/openapi/master/openapi/spec3.json) has a full list of event types. I ended up using that to add the missing types for treasury.

Released in [v2.2.0](https://github.com/kgajera/stripe-event-types/releases/tag/v2.2.0).
